src/footer/Footer.tsx

```diff
@@ -90,6 +90,9 @@ export default factory(function Footer({ middleware: { theme, i18n } }) {
 								<Link to="examples" classes={css.link}>
 									{messages.examples}
 								</Link>
+								<Link to="dojoswag" class={css.link}>
```

Comment:
I realize the examples link above it is wrong too, but this should be using the themed version of the css: `themedCss.link`.

Reply:
Done.